src/sections/Benefits/Benefits.tsx

```diff
@@ -28,7 +27,6 @@ const BenefitsContainer = styled(Stack).attrs({
 export const Benefits = () => (
   <BenefitsContainer id="benefits">
     <BenefitsList />
-    <NotComponentDriven />
```

Comment:
Our feeling is that this section is unnecessarily inflammatory (unless that's intentional?):

- Why aren't website builders component driven? You are building things out of components after all!

- You can do server-renderer development in a component driven way. Sure most companies aren't but it probably just irritates people to tell them you *can't*.

I reckon we should focus on the positives here; there's no need to throw shade around.